docs/client/getting-started.mdx

```diff
@@ -28,24 +26,59 @@ you may need to install these peer dependencies as well:
 ```
 </Info>
 
-</Step> <Step title="Initialize Client">
+</Step> <Step title="Initialize Library">
 
-To start integrating with the Skip Go API, initialize a `SkipClient` instance. This `skipClient` provides helper methods for common actions such as querying assets and chains, constructing routes, and executing transactions.
+To start integrating with the Skip Go API, you no longer initialize a `SkipClient` instance. Instead, you configure the library once and then import and use individual functions directly.
 
-```ts
-import { SkipClient } from "@skip-go/client";
+### Initialization Options
+
+The library can be initialized using `setClientOptions` or `setApiOptions`. Both functions accept `apiUrl` and `apiKey`.
+
+- **`setClientOptions(options)`:** Use this if you plan to use `executeRoute` or interact directly with blockchains. It configures your API credentials and lets you provide chain-specific settings like endpoints, Amino types, and registry types.
```

Comment:
i think the first sentence can just be:

Use this if you plan to use `executeRoute`. idk what the interact directly with blockchains means